src/EntryPoints/MediaWikiHooks.php

```diff
@@ -72,8 +72,13 @@ private static function addMappingUi( OutputPage $page, EntityId $entityId ): vo
 			 $page->getRequest()->getCheck( 'diff' )
 		);
 
-		$useCase = WikibaseRdfExtension::getInstance()->newShowMappingsUseCase( $presenter, $page->getUser() );
-		$useCase->showMappings( $entityId );
+		$repository = WikibaseRdfExtension::getInstance()->newMappingRepository( $page->getUser() );
+		$authorizer = WikibaseRdfExtension::getInstance()->newEntityMappingsAuthorizer( $page->getUser() );
+
+		$presenter->showMappings(
+			$repository->getMappings( $entityId, $page->getRevisionId() ?? 0 ),
```

Comment:
I'm doing the `0` here because that's what `RevisionLookup::getRevisionByTitle()` uses later. We can instead go with `null` here and change the rest of this to use `0` later if this is `null`.